src/components/layout/NavBarTooltip.css

```diff
@@ -0,0 +1,42 @@
+.navbar-tooltip {
+  position: absolute;
+  z-index: 1000;
+}
+
+.navbar-tooltip__wrapper {
+  position: relative;
+  background-color: var(--g3-color__white);
+  vertical-align: middle;
+  text-align: center;
```

Comment:
`text-align: center;` seems a bit weird to me, what do you think about `text-align: justify;`

Reply:
justify looks awkward because the tooltip is small and justify creates big spaces, i'll just use `left`